src/apps/opening-hours-editor/helper.ts

```diff
@@ -154,3 +156,18 @@ export const getDateString = (date: Date) => {
 export const getStringForDateInput = (date: Date) => {
   return dayjs(date).format("YYYY-MM-DD");
 };
+
+export const isOpeningHourWeeklyRepetition = (
+  event: EventImpl | EventInput
+) => {
+  if ("event" in event) {
+    return (
+      event.event.extendedProps?.repetition?.type ===
+      DplOpeningHoursListGET200ItemRepetitionType.weekly
+    );
+  }
```

Comment:
What is going on with the `if ("event" in event) {` check?

It might be me but I do not underrstand what is happening here.

Reply:
This is to make the function work with both `EventImpl` and `EventInput`. They are almost the same, but `EventInput` has an extra layer that is `event`.